Added a user snippet to block pings in tag arguments

ILikePlayingGames opened the pull request:
I just had multiple people spam pings with bot commands in my server recently and I realized I had some Carl tags that could also be affected. I came up with this line that prevents people from pinging others with Carl-bot using tag arguments.

ILikePlayingGames committed:
Added block pings in tag arguments user snippet

ILikePlayingGames committed:
Moved snippet to end of section, added missing comma

asty8926 commented:
If you think it can be useful to other users as well, I will merge the pull request.
However, I'd like to emphasize that "break" blocks do not prevent command blocks in the code from executing and only care about the final output. So instead of a break block, I would personally create a named condition (a condition that returns true or false in a variable that has a human-readable name), add a "blank" variable and make everything conditional in the tag rather than only add a break block.
While your suggestion is good when there is only an output, I feel like it could lead to errors for those who may not know that break blocks don't prevent command blocks from executing.
If you think you can find a solution to that, and/or add it in the description of the snippet, I think it would be even better.

Thank you for participating though, really appreciate it!

ILikePlayingGames commented:
Thank you for your feedback! I forgot about that, I'll change it to a condition.

ILikePlayingGames committed:
Updated block ping snippet

The snippet now blocks command blocks from executing and embed builder
embeds from sending. It also blocks pings to roles now. Thank you
Asty, asportnoy, Elise, Raffael, and Valkyrie for sharing tips and
code that made this possible!
